playwright/sync_api/_generated.py

```diff
@@ -12777,7 +12840,7 @@ def hover(
         """Locator.hover
 
         This method hovers over the element by performing the following steps:
-        1. Wait for [actionability](./actionability.md) checks on the element, unless `force` option is set.
+        1. Wait for [actionability](../actionability.md) checks on the element, unless `force` option is set.
```

Comment:
I assume relative links are expected

Reply:
broken links get fixed separately in https://github.com/microsoft/playwright-python/pull/1241.